servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -303,19 +314,25 @@ private boolean isRecordUpdatedWithNotificationTimeAndServer(String lastNotified
     }
 
     private List<Item> getUnrefreshedCertRecordsByDate(String provider, Index index, long yesterday, String unrefreshedCertDate) {
-        QuerySpec spec = new QuerySpec()
-                .withKeyConditionExpression("currentDate = :v_current_date")
-                .withFilterExpression("provider = :v_provider AND (attribute_not_exists(lastNotifiedTime) OR lastNotifiedTime < :v_last_notified)")
-                .withValueMap(new ValueMap()
-                        .withString(":v_current_date", unrefreshedCertDate)
-                        .withNumber(":v_last_notified", yesterday)
-                        .withString(":v_provider", provider));
-
-        ItemCollection<QueryOutcome> outcome = index.query(spec);
-        List<Item> items = new ArrayList<>();
-        for (Item item : outcome) {
-            items.add(item);
+        try {
+            QuerySpec spec = new QuerySpec()
+                    .withKeyConditionExpression("currentDate = :v_current_date")
```

Comment:
If an error is thrown for a specific combination of provider + date, we'll print it and continue so it won't affect other records.